tutorials/whisper_e2e.py

```diff
@@ -233,8 +233,8 @@ def postprocessing(token_ids, hf_processor):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument("-a", "--audio", required=True, help="Path to audio file")
-    parser.add_argument("-m", "--model", required=True, help="Path to custom export of Whisper with beam search")
+    parser.add_argument("-a", "--audio", required=True, help="Audio file input used to export model")
```

Comment:
more specifically, the audio file is to test the model generated, exporting doesn't really need an audio file